scripts/build.md

```diff
@@ -1,48 +0,0 @@
-# Building the Package
```

Comment:
Removing this because: 
* The part about the Install can be found in the IO docs
* The part about sphinx is not interesting from a user point of view externally and captured in Conf internally